### PR TITLE
SignUP UI added with functionality

### PR DESCRIPTION
**Pull request by amitsaini4556**

### CODE REVIEW

File: facultyRecruitment/userAccounts/forms.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+from validate_email import validate_email
+
+from .models import *
+
+class CreateUserForm(UserCreationForm):
+	class Meta:
+		model = User
+		fields = ['username', 'email', 'password1', 'password2']
+
+	def clean_email(self):
```
Comment (MuskanPaliwal) [inline facultyRecruitment/userAccounts/forms.py:14]: Just wondering, why is this named `clean_email`?

File: facultyRecruitment/userAccounts/forms.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+		email = self.cleaned_data.get('email')
+		if validate_email(email,verify=True) == True:
```
Comment (MuskanPaliwal) [inline facultyRecruitment/userAccounts/forms.py:16]: the `validate_email` is a function, right? and it returns a boolean value. I would like to suggest that the function should completely validate the email and return the error(`ValidationError`) or a response.

**Review by MuskanPaliwal — CHANGES_REQUESTED** [review]

File: facultyRecruitment/userAccounts/forms.py (hunk 2 above)
Comment (MuskanPaliwal) [reply facultyRecruitment/userAccounts/forms.py:16]: oh, it is an external library function.

**Review by MuskanPaliwal — APPROVED** [review]

File: facultyRecruitment/userAccounts/forms.py (hunk 1 above)
Comment (amitsaini4556) [reply facultyRecruitment/userAccounts/forms.py]: we can't change the method prefix 'clean'. 
In Django, It's the predefined syntax for validating parameters so we can't override.

**Pull request merged**